Misanthropic-Mistletoes/reactive-nutshell
# Readme

# Description

Check that the README.md is updated with download instructions and the database.json.example is updated.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

# Testing Instructions
1) `git fetch --all`
2) `git checkout readme`
3) See if README is updated and database.json is updated



# Checklist:
